modules/music_album.rb

```diff
@@ -0,0 +1,18 @@
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify, published_date, archived: true)
+    super(published_date, archived: archived)
+    @on_spotify = on_spotify
+  end
+
+  def to_json(*_args)
+    { 'on_spotify' => @on_spotify, 'published_date' => @published_date }
+  end
+
+  private
+
+  def can_be_archived?
+    (Time.now.year - @published_date) > 10 && @on_spotify != true
+  end
```

Comment:
- [ ] it would be better instead of calculating the time again you can call the parent's method by adding super that will be more clean and you will avoid DRY
```ruby
    def can_be_archived?
      super && @on_spotify == true
    end
```